build-system/tasks/e2e/selenium-webdriver-controller.js

```diff
@@ -82,6 +90,29 @@ class SeleniumWebDriverController {
     };
   }
 
+  async waitForElementProperty(handle, property, value) {
```

Comment:
I don't think this is the right solution, since the controller already handles this using the `ControllerPromise` and the overridden `expect`. We can implement this by passing these messages down to `driver.wait` call in the `waitFor` function. What do you think of that? 

Having `waitForBlah` semantics in the test code doesn't communicate the same intention that asserting expectations communicate. The API also becomes unclear, because there are two methods, `getElementProperty` and `waitForElementProperty`. We can still implement the improved messages without changing the API of the controller.

Reply:
Sounds good, will revisit these new methods.